matyhtf/framework/CVE-2021-43676.yaml

```diff
@@ -0,0 +1,8 @@
+title: Path manipulation
+link: https://github.com/advisories/GHSA-mh9j-v6mq-pfch
+cve: CVE-2021-43676
+reference: composer://matyhtf/framework
+branches:
+    '3.0':
+        time: null
+        versions: [ '<=3.0.5' ]
```

Comment:
we generally use `<3.0.6`, mentioning the version that provides the patch rather than the previous version.